modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/incoming/IncomingSnapshotCopier.java

```diff
@@ -157,9 +162,9 @@ public void join() throws InterruptedException {
     public void cancel() {
         canceled = true;
 
-        LOG.info("Copier is canceled for partition [partId={}, tableId={}]", partId(), tableId());
+        LOG.info("Copier is canceled for partition [{}]", createPartitionInfo());
 
-        CompletableFuture<?> fut = future;
+        CompletableFuture<?> fut = rebalanceFuture;
```

Comment:
Does it work? Do we have a test that would show that "cancel" works when there are no messages received by the node?

Reply:
Discussed personally, will work.